shared/gh/js/bootstrap.calendar.js

```diff
@@ -528,8 +594,20 @@ define(['gh.core', 'moment', 'clickover'], function(gh, moment) {
         $('#gh-calendar-toolbar-terms button').on('click', changeTerm);
         // Change the calendar's view
         $('#gh-calendar-toolbar-views button').on('click', changeView);
+
+        // Return the calendar's current view
+        $(document).on('gh.calendar.getCurrentView', function(evt, callback) {
+            return callback(getCurrentView());
```

Comment:
Could you add a line of docs to the `getCurrentView()` function that explains with an example what the return looks like?